ale/drivers/mes_drivers.py

```diff
@@ -225,6 +225,21 @@ class MessengerMdisIsisLabelNaifSpiceDriver(IsisLabel, NaifSpice, Framer, Driver
     Driver for reading MDIS ISIS3 Labels. These are Labels that have been ingested
     into ISIS from PDS EDR images but have not been spiceinit'd yet.
```

Comment:
This comment is misleading in that it makes it sound like we can't re-spiceinit a cube, or am I misreading it?

Reply:
I've re-worded this so it's clearer. This driver can work with spiceinit'd cubes but it will ignore any attached spice data. You need a different driver to work with spiceinit'd data.